README.md

```diff
@@ -421,6 +422,7 @@ Following options are available for the JS Obfuscator:
     --debug-protection-interval <boolean>
     --disable-console-output <boolean>
     --domain-lock '<list>' (comma separated)
+    --domain-dest 'url' <string>
```

Comment:
Please, add this option to the CLI: https://github.com/erikdubbelboer/javascript-obfuscator/blob/domain-dest/src/cli/JavaScriptObfuscatorCLI.ts

Reply:
Done.